Competition2018/src/main/java/competition/subsystems/drive/commands/RotateToHeadingCommand.java

```diff
@@ -0,0 +1,51 @@
+package competition.subsystems.drive.commands;
+
+import xbot.common.command.BaseCommand;
+import xbot.common.injection.wpi_factories.CommonLibFactory;
+import xbot.common.math.PIDFactory;
+import xbot.common.math.PIDManager;
+import xbot.common.subsystems.drive.control_logic.HeadingModule;
+import com.google.inject.Inject;
+import competition.subsystems.drive.DriveSubsystem;
+
+public class RotateToHeadingCommand extends BaseCommand{
+
+    double goal;
+    double power;
+    double heading;
+    final CommonLibFactory clf;
+    final PIDManager pid;
+    HeadingModule headingModule;
+    DriveSubsystem drive;
+    
+    @Inject
+    public RotateToHeadingCommand(CommonLibFactory clf, PIDFactory pf,  DriveSubsystem drive) {
+        this.clf = clf;
+        this.drive = drive;
+        pid = pf.createPIDManager("Goal Heading", 1, 0, 0);
+        pid.setErrorThreshold(0.1);
```

Comment:
Since the HeadingModule is normalized to -1, 1, an error threshold of 0.1 corresponds to an allowed error of 18 degrees. That's pretty high!

Reply:
changed error threshold to .01